src/Examples/Akka.Hosting.SqlSharding/Program.cs

```diff
@@ -12,6 +12,53 @@
 
 var builder = WebApplication.CreateBuilder(args);
 
+#if USE_OPTIONS
+builder.Services.AddAkka("MyActorSystem", configurationBuilder =>
+    {
+        var localConn = builder.Configuration.GetConnectionString("sqlServerLocal");
+        var shardingConn = builder.Configuration.GetConnectionString("sqlServerSharding");
+
+        var shardingJournalOptions = new SqlServerJournalOptions(isDefaultPlugin: false)
```

Comment:
Each options have their own set of adapters, if two persistence journal have the same adapters, each would be need to be set separately, or you can pass the reference to the other, ie. `journalOpt2.Adapters = journalOpt1.Adapters`. Or you can create an adapter builder instance and assign that to both.
The only caveat is that since it is an instance reference, you can't assign it to one and modify the other. We can change this behaviour to do an immutable copy on assignment instead.